src/CompanyService.Validation/CompanyUser/EditCompanyUserRequestValidator.cs

```diff
@@ -3,17 +3,20 @@
 using System.Threading.Tasks;
 using FluentValidation;
 using FluentValidation.Validators;
+using LT.DigitalOffice.CompanyService.Data.Interfaces;
 using LT.DigitalOffice.CompanyService.Models.Dto.Requests.CompanyUser;
 using LT.DigitalOffice.CompanyService.Validation.CompanyUser.Interfaces;
 using LT.DigitalOffice.Kernel.Validators;
-using Microsoft.AspNetCore.JsonPatch;
+using LT.DigitalOffice.Models.Broker.Enums;
 using Microsoft.AspNetCore.JsonPatch.Operations;
 
 namespace LT.DigitalOffice.CompanyService.Validation.CompanyUser
 {
   public class EditCompanyUserRequestValidator : BaseEditRequestValidator<EditCompanyUserRequest>, IEditCompanyUserRequestValidator
   {
-    private async Task HandleInternalPropertyValidation(
+    private readonly IContractSubjectRepository _contractSubjectRepository;
```

Comment:
Field 'EditCompanyUserRequestValidator._contractSubjectRepository' is never assigned to, and will always have its default value null